apps/mobile/package.json

```diff
@@ -29,7 +30,9 @@
     "lingui:extract": "lingui extract --clean",
     "lingui:extract:watch": "lingui extract --watch",
     "lingui:watch": "concurrently  \"pnpm lingui:extract:watch\"  \"pnpm lingui:compile:watch\"  ",
-    "preios": "pnpm run check:cocoapods && pnpm run firebase:ios",
+    "postinstall": "if [ \"$CI\" = \"1\" ]; then pnpm -w build:mobile; fi",
+    "preandroid": "pnpm run firebase && npx expo prebuild",
```

Comment:
Shouldn't need the `npx expo prebuild` step here, if it needs to be run the `expo run` commands should run it i believe? 

A bit of redundancy is fine with me, and maybe running it frequently kind of helps us ensure it's working as intended.